eng/dtdl/adr-base-service.json

```diff
@@ -1944,6 +1981,12 @@
             "elementSchema": "dtmi:com:microsoft:akri:EventStreamDestination"
           }
         },
+        {
+          "@type": "Field",
+          "name": "description",
+          "description": "Human-readable description of the asset.",
+          "schema": "string"
```

Comment:
I don't know if this matters, but we often use localized strings for "description" and "displayName", so this might be better as a Map from string (language codes) to string (language-specific description strings), rather than merely a string.

Reply:
The DTDL updates are coming from the Service, so we shouldn't be modifying them at all in our repo. Although this is good feedback, feel free to surface it to the ADR team to think about for the next release. Going to resolve these comments for now since they wouldn't take effect in this PR unless higher up takes this change